Templates/CSharp/EdgeDriverTemplate/ProjectTemplates/CSharp/Test/EdgeDriverTest.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Edge;
+
+namespace $safeprojectname$
+{
+    [TestClass]
+    public class EdgeDriverTest
+    {
+        // In order to use web driver of Edge, it is required to download and install the Edge Web Driver from
+        // https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
+        // Then add it to your PATH enviroment variable and run it at the background.
```

Comment:
We should request Clay Martin and co. to review the code in this file including the text of this comment.

Reply:
Something like this may be better -
`In order to run the below tests, download and install Microsoft WebDriver from http://***, add the folder containing the installed xyz.exe to your PATH enviornment variable and launch xyz.exe from a command prompt / from an admin command prompt.`

Couple of things:
1. We should ask Clay Martin if they have a fwdlink to the above website so that the link will continue to work even if the website moves to a different location in the future.
2. Please mention name of the executable that the user needs to run.
3. If the executable needs to be run in an admin prompt, please specify that too.

Reply:
I will send an email to ask them to have a review at this.